### PR TITLE
stat: print information about all buckets when only alias are set

### PR DESCRIPTION
**Pull request by dormanze**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Motivation and Context
When view information about all buckets at a time, details such as the bucket usage cannot be printed.
image
Buckets should be treated as special directories because we have metrics for bucket.

## How to test this PR?
mc stat myminio --verbose
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by shtripat** [comment]
Can you please add `Motivation and Context` for the change?

File: cmd/stat-main.go [hunk 1]
```diff
@@ -71,20 +72,23 @@ EXAMPLES:
   1. Stat all contents of mybucket on Amazon S3 cloud storage.
      {{.Prompt}} {{.HelpName}} s3/mybucket/
 
-  2. Stat all contents of mybucket on Amazon S3 cloud storage on Microsoft Windows.
+  2. Stat all contents of all bucket on Amazon S3 cloud storage.
```
Comment (shtripat) [inline cmd/stat-main.go:75]:
```suggestion
  2. Stat all contents of all buckets on Amazon S3 cloud storage.
```

**Review by shtripat — COMMENTED** [review]
Changes look fine.
But for a huge deployment with lot of buckets the output would be huge. Is the output like streaming of data?
What would be the case for `--json` output?

**Comment by klauspost** [comment]
Quite nice change. It does however feel very verbose, and listing 50 buckets will make it pretty useless as @shtripat points out.

Could we add the extended information behind a `-v`/`--verbose` flag.

As a bonus, could you make it so `1970` (unix zero) dates aren't shown?

**Review by klauspost — APPROVED** [review]
LGTMgggd

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**